tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -629,21 +622,64 @@ def _create_multi_device_plugin(self, log=True):
 
         if num_devices != len(num_per_device_requests):
             raise ConfigError('num requests for all {} should be specified'.format(num_devices))
-        device_strings = [
-            '{device}({nreq})'.format(device=device, nreq=nreq)
-            for device, nreq in zip(device_list, num_per_device_requests)
-        ]
-        self.plugin = ie.IEPlugin('MULTI:{}'.format(','.join(device_strings)))
-        self._num_requests = sum(num_per_device_requests)*2
+        self._num_requests = sum(num_per_device_requests) * 2
         if log:
-            print_info('Loaded {} plugin version: {}'.format(self.plugin.device, self.plugin.version))
+            self._log_versions()
             print_info('Request number for each device:')
             for device, nreq in zip(device_list, num_per_device_requests):
                 print_info('    {} - {}'.format(device, nreq))
 
-    def _create_network(self, input_shapes=None):
-        assert self.plugin, "create_ie_plugin should be called before _create_network"
+    def set_device_config(self, device_config):
+        device_specific_configuration = read_yaml(device_config)
+        if not isinstance(device_specific_configuration, dict):
+            raise ConfigError('device configuration should be a dict-like')
+        self.ie_core.set_config(device_specific_configuration, self.device)
+
+    def _log_versions(self):
+        versions = self.ie_core.get_versions(self._device)
+        print_info("Loaded {} plugin version:".format(self._device))
+        for device_name, device_version in versions.items():
+            print_info("    {device_name} - {descr}: {maj}.{min}.{num}".format(
+                device_name=device_name, descr=device_version.description, maj=device_version.major,
+                min=device_version.minor, num=device_version.build_number
+            ))
+
+    def auto_num_requests(self):
```

Comment:
I think you need to use `GetMetric` with `OPTIMAL_NUMBER_OF_INFER_REQUESTS` to determine number of infer requests.

Reply:
Thanks, i have plan to integrate metrics usage in AC, but it will be done later? in separated PR? because It is too braking changes for the tool and required additional validation on full models scope

Reply:
@ilya-lavrenov, probably, I need your help with it...
If I right understand, you suggest get metric from ExecutableNetwork. 
But due to some IE Python limitations, I can not use ExecutableNetwork.createInferRequest() for InferRequest creatintion (this method does not implemented), they can be created only with ExecutableNetwork during IECore().loadNetwork().
It means that I need:
1. create ExecutableNetwork
2. get optimal number of infer requests
3. recreate ExecutableNetwork with requred number of requests.
Is it correct? Is there no way without ExecutableNetwork recreation? 
(for some models load  is time consuming operation, especially if we speak about INT8...)